.github/workflows/release-github.yml

```
@@ -20,6 +20,7 @@ on:
 
 jobs:
   build:
+    name: "Oras build & release"
```

This would be better
```suggestion
  release:
```